go/interpreter/lfvm/instructions_test.go

```diff
@@ -1147,3 +1147,151 @@ func TestGenericCreate_MaxInitCodeSizeIsNotCheckedBeforeShanghai(t *testing.T) {
 		})
 	}
 }
+
+func TestInstructions_EIP2929_staticGasCostForBerlinAndAfterIsZero(t *testing.T) {
```

Comment:
```suggestion
func TestInstructions_EIP2929_StaticGasCostIsZero(t *testing.T) {
```
EIP2929 already says Berlin or later